packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -78,6 +76,8 @@ export class ChannelStorageService implements IChannelStorageService {
 	}
 
 	private async getIdForPath(path: string): Promise<string> {
-		return this.flattenedTree[path];
+		// TODO why are we non null asserting here?
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		return this.flattenedTree[path]!;
```

Comment:
If you change the return type of the function to return undefined, you can get rid of the null assert. But of course that will mean callers have to handle the undefined.

Reply:
Might be worth doing this now since there's only the single callsite in the file.